public/scripts/openai.js

```diff
@@ -161,13 +165,26 @@ export const chat_completion_sources = {
     CLAUDE: 'claude',
     SCALE: 'scale',
     OPENROUTER: 'openrouter',
+    AI21: 'ai21',
 };
 
+const prefixMap = selected_group ? {
```

Comment:
This should be an area of note and played around with. I'm unsure of what the best prefixes for messages with this model, but I think charname: for messages and OOC: or prompt: for system instructions are the best.